### PR TITLE
Update DeviceInfoQr.tsx

### PR DESCRIPTION
**Pull request by dobbz**
Get full info of device specifications in QR code.

### CODE REVIEW

**Review by kevinmainairungu — APPROVED** [review]
@mutterback please check this video recording of this code change. This is what we need to fetch from the devices. We might need to edit more so that the .toJson output can be useful when uploading to power apps.

**Comment by kevinmainairungu** [comment]
@mutterback these are the changes we wanted to be added to the feature/dev branch. It has the correct data we need for collecting device specs. 
@brettins could this changes be merged into feature/dev

File: .vscode/settings.json [hunk 1]
```diff
@@ -25,4 +25,5 @@
   "javascript.preferences.quoteStyle": "double",
   "typescript.preferences.quoteStyle": "double",
   "prettier.singleQuote": false,
+  "python.pythonPath": "c:\\Python38\\python.exe",
```
Comment (brettins) [inline .vscode/settings.json:28]: This is windows specific & machine specific and doesn't belong in a commit

File: src/renderer/library/components/settings/DeviceInfoQr.tsx [hunk 2]
```diff
@@ -34,8 +34,8 @@ class DeviceInfoQr extends React.Component<IProps, IState> {
     let qrCodeToShow
     if (deviceInfo && deviceEkitabuUuid != '') {
       const qrCodeDeviceInfo = {
-        windowsSerialNumber: deviceInfo.osInfo.serial,
-        hardwareSerialNumber: deviceInfo.sysInfo.uuid,
+        windowsSerialNumber: deviceInfo.osInfo,
```
Comment (brettins) [inline src/renderer/library/components/settings/DeviceInfoQr.tsx:37]: these variable names are no longer correct if we're getting the entire `osInfo` and `sysInfo`,

**Review by brettins — CHANGES_REQUESTED** [review]

**Comment by brettins** [comment]
@kevinmainairungu all of the translation files have been moved out of "en.md" type files and into `en.xhtml` files in the new version of Readium, so most of these translations won't work anymore.

File: src/renderer/library/components/settings/DeviceInfoQr.tsx (hunk 2 above)
Comment (kevinmainairungu) [reply src/renderer/library/components/settings/DeviceInfoQr.tsx:37]: Changed variables names

File: .vscode/settings.json (hunk 1 above)
Comment (kevinmainairungu) [reply .vscode/settings.json:28]: added to .gitignore.

**Pull request closed without merging**